protocol/message.go

```diff
@@ -36,6 +36,7 @@ type Request struct {
 // The response to a successful request is a DirectoryProof with a TB for
 // the requested username and public key.
 type RegistrationRequest struct {
+	Epoch                  uint64
```

Comment:
What epoch is the client supposed to use here? Is the assumption that the client has fetched the latest STR before it even registers the name for the first time?